src/test/java/bio/terra/janitor/integration/TrackResourceIntegrationTest.java

```diff
@@ -262,6 +281,70 @@ public void subscribeAndCleanupResource_googleBigQueryTable() throws Exception {
     assertTrue(bigQueryCow.delete(datasetName));
   }
 
+  @Test
+  public void subscribeAndCleanupResource_googleProject() throws Exception {
+    String projectId = randomProjectId();
+
+    Operation operation =
+        resourceManagerCow
+            .projects()
+            .create(new Project().setProjectId(projectId).setParent(parentResourceId))
+            .execute();
+    OperationCow<Operation> operationCow = resourceManagerCow.operations().operationCow(operation);
+    operationCow =
+        OperationUtils.pollUntilComplete(
+            operationCow, Duration.ofSeconds(5), Duration.ofSeconds(30));
+    assertTrue(operationCow.getOperation().getDone());
+    assertNull(operationCow.getOperation().getError());
```

Comment:
make a helper function for creating a project. The details of how the project are created are not interesting to the test, so they don't need to be inlined, and we repeat them in the next test as well.